tools/DocsLinter/Tests/SimplifiedMarkdownDocTests.cs

```diff
@@ -8,13 +7,58 @@ namespace DocsLinter.Tests
     [TestFixture]
     public class SimplifiedMarkdownDocTests
     {
+        private static readonly string Title = "Title";
+
+        private SimplifiedMarkdownDoc GetSimplifiedMarkdown(RemoteLink remoteLink)
+        {
+            var markdownUrl = $"[{Title}]({remoteLink.Url})";
+            var markdownDoc = Markdown.Parse(markdownUrl);
+            return new SimplifiedMarkdownDoc(markdownDoc);
+        }
+
+        private RemoteLink CreateRemoteLink(string url)
+        {
+            return new RemoteLink(new LinkInline(url, Title));
+        }
+
+        [Test]
+        public void Link_starting_with_http_is_valid()
+        {
+            var url = CreateRemoteLink("http://www.google.com");
+            var markdown = GetSimplifiedMarkdown(url);
+
+            Assert.IsTrue(markdown.Links.Count == 1);
+            Assert.IsTrue(markdown.Links.Contains(url));
+        }
+
+        [Test]
+        public void Link_starting_with_https_is_valid()
+        {
+            var url = CreateRemoteLink("https://www.google.com");
+            var markdown = GetSimplifiedMarkdown(url);
+
+            Assert.IsTrue(markdown.Links.Count == 1);
+            Assert.IsTrue(markdown.Links.Contains(url));
+        }
+
+        [Test]
+        public void Link_starting_with_site_name_is_ignored()
```

Comment:
Did we decide to ignore these links then?

Reply:
it matches the behaviour of `Uri.IsWellFormedUriString` https://docs.microsoft.com/en-us/dotnet/api/system.uri.iswellformeduristring?view=netcore-2.2#System_Uri_IsWellFormedUriString_System_String_System_UriKind_